gammapy/scripts/image_pipe.py

```diff
@@ -211,6 +224,44 @@ def excess_image(self):
         total_excess.data = self.maps["counts"].data - self.maps["bkg"].data
         self.maps["excess"] = total_excess
 
+    def make_mean_psf(self, region_center, spectral_index=2.3):
```

Comment:
Changes:
- `make_mean_psf` -> `make_psf`
- `region_center` -> `pos`
- `spectral_index` can be removed
- Update docstring
- Remove if-else. Either don't do any check, or raise an error. Normal code (what's now in the else block) shouldn't be indented.